cmd/context/use.go

```diff
@@ -226,17 +226,17 @@ func getContext(ctxOptions *Options) (string, error) {
 	return oktetoContext, nil
 }
 
-func setSecrets(secrets []types.Secret) {
-	for _, secret := range secrets {
-		value, exists := os.LookupEnv(secret.Name)
+func exportPlatformVariablesToEnv(variables []env.Var) {
+	for _, v := range variables {
+		value, exists := os.LookupEnv(v.Name)
 		if exists {
-			if value != secret.Value {
-				oktetoLog.Warning("$%s secret is being overridden by a local environment variable by the same name.", secret.Name)
+			if value != v.Value {
+				oktetoLog.Warning("$%s User or Admin variable defined in the Okteto UI is being overridden by a local environment variable with the same name.", v.Name)
```

Comment:
@pchico83 can you help me unblock this PR and agree on a msg? I've updated to `User or Admin variable` instead of `Platform Variables`. WDYT?

Reply:
@andreafalzetti it could also be a variable in `.env` right? I would just say:
```
Okteto Variable '%s' is overridden by a local environment variable with the same name
```